fw_jump relocates the DTB to FW_JUMP_FDT_ADDR before entering the kernel.
Address is absolute, it controls minimum amount of RAM.

Kernel now at ~4.8MB, so at the +5MB offset we still have a bit of buffer.

--- opensbi-1.3/platform/generic/objects.mk
+++ opensbi-1.3/platform/generic/objects.mk
@@ -28,7 +28,7 @@
   # This needs to be 2MB aligned for 64-bit system
   FW_JUMP_ADDR=$(shell printf "0x%X" $$(($(FW_TEXT_START) + 0x200000)))
 endif
-FW_JUMP_FDT_ADDR=$(shell printf "0x%X" $$(($(FW_TEXT_START) + 0x2200000)))
+FW_JUMP_FDT_ADDR=$(shell printf "0x%X" $$(($(FW_TEXT_START) + 0x500000)))
 FW_PAYLOAD=y
 ifeq ($(PLATFORM_RISCV_XLEN), 32)
   # This needs to be 4MB aligned for 32-bit system
